docs/AGENTS.md

```
@@ -0,0 +1,36 @@
+# The Agents of the Loom
+
+This document defines the roles of the various "Intelligences" within the RealmWalker architecture.
+
+## 1. The Weaver ("God")
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)